tests/conftest.py

```diff
@@ -945,7 +945,17 @@ def finalizer():
 
         for instance in instances:
             try:
-                instance.delete()
+                instance.delete(wait=False)
```

Comment:
Should this be after deleting radosnamespaces ?

Reply:
As per interim fix added by leela -
when cbp is marked for deletion in that case only Radosnamespace will get deleted otherwise it will get reconsiled. 
Ones  Radosnamespace get deleted cbp which is marked for deletion will  also get deleted.